specification/azurefleet/resource-manager/Microsoft.AzureFleet/stable/2024-11-01/azurefleet.json

```diff
@@ -2467,7 +2470,7 @@
         },
         "localStorageInGiB": {
           "$ref": "#/definitions/VMAttributeMinMaxDouble",
-          "description": "The range of local storage in GB specified from Min to Max."
+          "description": "LocalStorageSupport should be set to \"Included\" or \"Required\" to use this VMAttribute. \nIf localStorageSupport is \"Excluded\", this VMAttribute can not be used.\" should be added to the description for localStorageInGiB."
```

Comment:
Please remove "**should be added to the description for localStorageInGiB."** from description.

Reply:
Removed from tsp, the json will get updated.